### PR TITLE
Feature for revision

### PR DESCRIPTION
**Pull request by bernabasy**
added file
-index.html
-styles.css
-readme

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        </p>
+      </div>
+      <div class="ratings">
+        <div class="rating-items item1">
+          <div class="images">
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
```
Comment (bernabasy) [inline index.html:34]: images file is not added to your project, so it's good to upload images before using them in your file, so create a folder and upload all images to the folder

File: index.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            <p>Rated 5 Stars in Reviews</p>
+          </div>
+        </div>
+        <div class="rating-items item2">
+          <div class="images">
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
```
Comment (bernabasy) [inline index.html:46]: as I said above images file is not added to your project, so it's good to upload images before using them in your file, so create a folder and upload all images to the folder

File: index.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+            <p>Rated 5 Stars in Report Guru</p>
+          </div>
+        </div>
+        <div class="rating-items item3">
+          <div class="images">
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
+            <img src="/images/icon-star.svg" alt="" />
```
Comment (bernabasy) [inline index.html:58]: images file is not added to your project, so it's good to upload images before using them in your file, so create a folder and upload all images to the folder

File: index.html [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+          </div>
+          <p>Rated 5 Stars in BestTech</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="cards-container">
+      <div class="card-items card-item1">
+        <div class="image-container">
+          <img class="card-img" src="/images/image-colton.jpg" alt="" />
```
Comment (bernabasy) [inline index.html:68]: its also same comment on images, images file is not added to your project, so it's good to upload images before using them in your file, so create a folder and upload all images to the folder

File: index.html [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+        <p class="description">
+          "We needed the same printed design as the one we had ordered a week
+          prior. Not only did they find the original order, but we also received
+          it in time. Excellent!"
+        </p>
+      </div>
+      <div class="card-items card-item3">
+        <div class="image-container">
+          <img class="card-img" src="/images/image-anne.jpg" alt="" />
+          <p class="title">
```
Comment (bernabasy) [inline index.html:97]: as i said on the comment above, images file is not added to your project, so it's good to upload images before using them in your file, so create a folder and upload all images to the folder

File: style.css [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+  --color-neutral-2: hsl(300, 24%, 96%);
+  --color-white: hsl(0, 0%, 100%);
+}
+body {
+  width: 70%;
+  max-width: 1440px;
+  height: 100vh;
+  background-color: var(--color-white);
+  background-image: url(/images/bg-pattern-top-desktop.svg),
+    url(/images/bg-pattern-bottom-desktop.svg);
```
Comment (bernabasy) [inline style.css:20]: background-image is not uploaded to your project, please create the folder and upload images to it, or you can use  an image URL from online access.

**Review by bernabasy — COMMENTED** [review]
### changes required ♻️ 
- good comment
- beast HTML & CSS  practices
- you almost did it don't worry, it need little steeps 🦾
-pleas configure linters , that makes you project easier to work, to find bugs and fix it.
-pleas check my comments  below and  make changes to your project. then your project works correctly
thanks👍

**Review by emmiiorji — APPROVED** [review]
Hi @bernabasy, 

# Status: Approved ✅

Your project is complete! You've done greatly at reviewing your past project. There is nothing else to say other than... :shipit: Congratulations! 🎉 

congratulations


## Good Highlights ✅

- You suggested changes to make the code better
- Good use of inline comments
- Reasons were given for suggested changes

## Optional suggestions

- Please, as much as possible, avoid repeating the same comments in several places. If they're similar mistakes, you could refer to a sample and mention how there are other appearances of the same error. This will avoid unnecessary repetitions.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**